app/controllers/testimonials_controller.rb

```diff
@@ -0,0 +1,53 @@
+class TestimonialsController < ApplicationController
```

Comment:
Please check the description of the ticket, because it doesn't specify any endpoint, and you have to put  the controller into the namespace `api/v1`.